### ISSUE TITLE
Cannot create a new database with language: Spanish(BO)

### ISSUE THREAD

**Issue by icoro** [odoo/odoo#27992]
Impacted versions: v12

Steps to reproduce:
Create a new database with Spanish(BO) as language

Current behavior:
This error is thrown:
```
2018-10-20 00:03:28,423 30022 INFO demo1 odoo.tools.translate: loading /Users/ivan/dev/odoo/v12/odoo/odoo/addons/base/i18n/es_BO.po 
2018-10-20 00:03:29,180 30022 ERROR demo1 odoo.sql_db: bad query: b" INSERT INTO ir_translation(name, lang, res_id, src, type, value, module, state, comments)\n                           SELECT name, lang, res_id, src, type, value, module, state, comments\n                           FROM tmp_ir_translation_import\n                           WHERE type IN ('selection', 'constraint', 'sql_constraint')\n                           ON CONFLICT (type, lang, name, md5(src)) WHERE type IN ('selection', 'constraint', 'sql_constraint')\n                            DO UPDATE SET (name, lang, res_id, src, type, value, module, state, comments) = (EXCLUDED.name, EXCLUDED.lang, EXCLUDED.res_id, EXCLUDED.src, EXCLUDED.type, EXCLUDED.value, EXCLUDED.module, EXCLUDED.state, EXCLUDED.comments);\n                       "
ERROR: there is no unique or exclusion constraint matching the ON CONFLICT specification
 
2018-10-20 00:03:29,228 30022 WARNING demo1 odoo.modules.loading: Transient module states were reset 
2018-10-20 00:03:29,229 30022 ERROR demo1 odoo.modules.registry: Failed to load registry 
Traceback (most recent call last):
  File "/Users/ivan/dev/odoo/v12/odoo/odoo/modules/registry.py", line 86, in new
    odoo.modules.load_modules(registry._db, force_demo, status, update_module)
  File "/Users/ivan/dev/odoo/v12/odoo/odoo/modules/loading.py", line 367, in load_modules
    report=report, models_to_check=models_to_check)
  File "/Users/ivan/dev/odoo/v12/odoo/odoo/modules/loading.py", line 233, in load_module_graph
    module.with_context(overwrite=overwrite)._update_translations()
  File "/Users/ivan/dev/odoo/v12/odoo/odoo/addons/base/models/ir_module.py", line 860, in _update_translations
    self.env['ir.translation'].load_module_terms(mod_names, filter_lang)
  File "/Users/ivan/dev/odoo/v12/odoo/odoo/addons/base/models/ir_translation.py", line 760, in load_module_terms
    tools.trans_load(self._cr, trans_file, lang, verbose=False, module_name=module_name, context=context)
  File "/Users/ivan/dev/odoo/v12/odoo/odoo/tools/translate.py", line 1015, in trans_load
    result = trans_load_data(cr, fileobj, fileformat, lang, verbose=verbose, module_name=module_name, context=context)
  File "/Users/ivan/dev/odoo/v12/odoo/odoo/tools/translate.py", line 1155, in trans_load_data
    irt_cursor.finish()
  File "/Users/ivan/dev/odoo/v12/odoo/odoo/addons/base/models/ir_translation.py", line 159, in finish
    """ % (self._model_table, self._table))
  File "/Users/ivan/dev/odoo/v12/odoo/odoo/sql_db.py", line 148, in wrapper
    return f(self, *args, **kwargs)
  File "/Users/ivan/dev/odoo/v12/odoo/odoo/sql_db.py", line 225, in execute
    res = self._obj.execute(query, params)
psycopg2.ProgrammingError: there is no unique or exclusion constraint matching the ON CONFLICT specification

2018-10-20 00:03:29,230 30022 ERROR demo1 odoo.service.db: CREATE DATABASE failed: 
Traceback (most recent call last):
  File "/Users/ivan/dev/odoo/v12/odoo/odoo/service/db.py", line 62, in _initialize_db
    registry = odoo.modules.registry.Registry.new(db_name, demo, None, update_module=True)
  File "/Users/ivan/dev/odoo/v12/odoo/odoo/modules/registry.py", line 86, in new
    odoo.modules.load_modules(registry._db, force_demo, status, update_module)
  File "/Users/ivan/dev/odoo/v12/odoo/odoo/modules/loading.py", line 367, in load_modules
    report=report, models_to_check=models_to_check)
  File "/Users/ivan/dev/odoo/v12/odoo/odoo/modules/loading.py", line 233, in load_module_graph
    module.with_context(overwrite=overwrite)._update_translations()
  File "/Users/ivan/dev/odoo/v12/odoo/odoo/addons/base/models/ir_module.py", line 860, in _update_translations
    self.env['ir.translation'].load_module_terms(mod_names, filter_lang)
  File "/Users/ivan/dev/odoo/v12/odoo/odoo/addons/base/models/ir_translation.py", line 760, in load_module_terms
    tools.trans_load(self._cr, trans_file, lang, verbose=False, module_name=module_name, context=context)
  File "/Users/ivan/dev/odoo/v12/odoo/odoo/tools/translate.py", line 1015, in trans_load
    result = trans_load_data(cr, fileobj, fileformat, lang, verbose=verbose, module_name=module_name, context=context)
  File "/Users/ivan/dev/odoo/v12/odoo/odoo/tools/translate.py", line 1155, in trans_load_data
    irt_cursor.finish()
  File "/Users/ivan/dev/odoo/v12/odoo/odoo/addons/base/models/ir_translation.py", line 159, in finish
    """ % (self._model_table, self._table))
  File "/Users/ivan/dev/odoo/v12/odoo/odoo/sql_db.py", line 148, in wrapper
    return f(self, *args, **kwargs)
  File "/Users/ivan/dev/odoo/v12/odoo/odoo/sql_db.py", line 225, in execute
    res = self._obj.execute(query, params)
psycopg2.ProgrammingError: there is no unique or exclusion constraint matching the ON CONFLICT specification

2018-10-20 00:03:29,238 30022 INFO demo1 odoo.modules.loading: loading 1 modules... 
2018-10-20 00:03:29,252 30022 INFO demo1 odoo.modules.loading: 1 modules loaded in 0.01s, 0 queries 
2018-10-20 00:03:29,253 30022 ERROR demo1 odoo.sql_db: bad query: b"SELECT * FROM ir_model WHERE state='manual'"
ERROR: relation "ir_model" does not exist
LINE 1: SELECT * FROM ir_model WHERE state='manual'
                      ^
 
2018-10-20 00:03:29,255 30022 WARNING demo1 odoo.modules.loading: Transient module states were reset 
2018-10-20 00:03:29,256 30022 ERROR demo1 odoo.modules.registry: Failed to load registry 
Traceback (most recent call last):
  File "/Users/ivan/dev/odoo/v12/odoo/odoo/modules/registry.py", line 60, in __new__
    return cls.registries[db_name]
  File "/Users/ivan/dev/odoo/v12/odoo/odoo/tools/func.py", line 69, in wrapper
    return func(self, *args, **kwargs)
  File "/Users/ivan/dev/odoo/v12/odoo/odoo/tools/lru.py", line 44, in __getitem__
    a = self.d[obj].me
KeyError: 'demo1'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/ivan/dev/odoo/v12/odoo/odoo/modules/registry.py", line 86, in new
    odoo.modules.load_modules(registry._db, force_demo, status, update_module)
  File "/Users/ivan/dev/odoo/v12/odoo/odoo/modules/loading.py", line 372, in load_modules
    registry.setup_models(cr)
  File "/Users/ivan/dev/odoo/v12/odoo/odoo/modules/registry.py", line 254, in setup_models
    env['ir.model']._add_manual_models()
  File "/Users/ivan/dev/odoo/v12/odoo/odoo/addons/base/models/ir_model.py", line 302, in _add_manual_models
    cr.execute('SELECT * FROM ir_model WHERE state=%s', ['manual'])
  File "/Users/ivan/dev/odoo/v12/odoo/odoo/sql_db.py", line 148, in wrapper
    return f(self, *args, **kwargs)
  File "/Users/ivan/dev/odoo/v12/odoo/odoo/sql_db.py", line 225, in execute
    res = self._obj.execute(query, params)
psycopg2.ProgrammingError: relation "ir_model" does not exist
LINE 1: SELECT * FROM ir_model WHERE state='manual'
                      ^

2018-10-20 00:03:29,422 30022 INFO demo1 odoo.sql_db: ConnectionPool(used=0/count=2/max=64): Closed 1 connections to 'sslmode=prefer dbname=demo1' 
2018-10-20 00:03:29,423 30022 INFO demo1 odoo.sql_db: ConnectionPool(used=0/count=1/max=64): Closed 1 connections to 'sslmode=prefer dbname=i9' 
2018-10-20 00:03:29,423 30022 INFO demo1 odoo.sql_db: ConnectionPool(used=0/count=0/max=64): Closed 1 connections to 'sslmode=prefer dbname=i9prod' 
2018-10-20 00:03:29,427 30022 INFO demo1 werkzeug: 127.0.0.1 - - [20/Oct/2018 00:03:29] "POST /web/database/create HTTP/1.1" 200 - 18419 7.779 6.669
Expected behavior:
```
Video/Screenshot link (optional):

**Comment by celm1990** [comment]
For Spanish/Ecuador(es_EC) and Spanish/Chile(es_CL) same error

**Comment by mart-e** [comment]
Hello,

It was already fixed at 57884ccdc8621f7c489a24e0bb89ac6b47763e12, please update your source code and you should no longer have this problem.
